packages/terra-site/src/examples/collapse/OpenCloseEventCollapse.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Collapse from 'terra-collapse/src/Collapse';
+
+const OpenCloseEventCollapse = () => (
+  <Collapse
+    closedButtonText="On Open and On Close Events"
+    onOpen={() => alert('Fired when collapse is opened')}
```

Comment:
I think we should keep with this practice, and I'm also leaning towards having the nightwatch test do that as well.